README.md

```diff
@@ -118,4 +118,15 @@ checks:
 		
 		# Check that a field does not exist:
 		not-has-key: flow
+
+  - shell:
+      # A simple shell check. If command exits non 0 the check
```

Comment:
suggest: If the command exists with a non-0 exit code, the check...

Reply:
s/exists/exits